var/spack/repos/builtin/packages/py-basemap/package.py

```diff
@@ -13,6 +13,8 @@ class PyBasemap(PythonPackage):
 
     homepage = "http://matplotlib.org/basemap/"
```

Comment:
Can you add a `url` attribute so that `spack versions` and `spack checksum` work properly?